tests/services/test_burger_menu_service.py

```diff
@@ -52,3 +53,36 @@ def test_get_burger_menu_model__with_supported_language__should_return_documents
         self.assertIsNot(len(burger_menu_model.documents), 0)
         for document in burger_menu_model.documents:
             self.assertEquals(document.document_link.language, SupportedLanguage.GERMAN)
+
```

Comment:
you now only test for spanish, should we test all languages?

Reply:
I think it's fine... the intention of the test is not to test Spanish but something that would have broken before (unicode)... I would include more weird characters if we see it makes sense

Reply:
@Josef-MrBeam I added hopefully all the weird symbols from the 4 languages 😄